src/vma/sock/sockinfo_udp.cpp

```diff
@@ -1844,6 +1850,8 @@ bool sockinfo_udp::rx_input_cb(mem_buf_desc_t* p_desc, void* pv_fd_ready_array)
 		pkt_info.socket_ready_queue_byte_count = m_p_socket_stats->n_rx_ready_byte_count;
 		if (m_n_tsing_flags & SOF_TIMESTAMPING_RAW_HARDWARE)
 			pkt_info.hw_timestamp = p_desc->path.rx.hw_timestamp;
+		if (p_desc->path.rx.sw_timestamp.tv_sec)
```

Comment:
1. It is not sufficient to base your if statement on tv_sec field only
2. Please use { } with if-statement (consistency with code styling)